pilot/cmd/pilot-agent/status/ready/stats.go

```diff
@@ -0,0 +1,72 @@
+// Copyright 2018 Istio Authors
```

Comment:
2019

Reply:
This file was actually moved, but git didn't recognize the move because the file changed so much.